app/src/main/res/layout/participant_primary_view.xml

```diff
@@ -1,10 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/participant_background"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <!-- RemoteParticipant layout without video track -->
```

Comment:
Moved this layout to the top as it provides a better view in the design window in Android Studio.